modules/serverless-traffic-management-kn.adoc

```diff
@@ -0,0 +1,63 @@
+[id="serverless-traffic-management-kn_{context}"]
+= Traffic management using the Knative CLI
```

Comment:
Because this is reference, consider "= Knative CLI syntax for traffic management"  (Or "traffic splitting", I have a comment elsewhere that hopefully you can limit to one term.)

The "using" prolly would be "by using" according to ISG, but that wording has a slight implication for a procedure and this is reference information.